src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueTest.java

```diff
@@ -40,7 +39,6 @@ void whenMeshInboundQueueMessageIsReceivedThenMessageIsHandled(SoftAssertions so
         final String content = parseTextMessage(message);
         final String expectedContent = new String(Files.readAllBytes(getFhirResource().getFile().toPath()));
 
-        softly.assertThat(message.getStringProperty("TransactionType")).isEqualTo(TransactionType.APPROVAL.name().toLowerCase());
         softly.assertThat(content).isEqualTo(expectedContent);
```

Comment:
No need for softly anymore.
```suggestion
        assertThat(content).isEqualTo(expectedContent);
```

Reply:
Have removed `softly` in 697ebb8